praw/objects.py

```diff
@@ -1474,6 +1478,26 @@ def clear_all_flair(self):
         else:
             return
 
+    def leave_contributor(self):
```

Comment:
Once the appropriate mixin exists, these functions can be dynamically created by adding two entries to `Subreddit._methods`.